scripts/es2k/setup_infra.sh

```diff
@@ -0,0 +1,272 @@
+    rm -rf $STRATUM_DIR/certs/client.key
+    rm -rf $STRATUM_DIR/certs/stratum.crt
+    rm -rf $STRATUM_DIR/certs/stratum.key
+    cp $BASE_DIR/tls/certs/infrap4d/* /usr/share/stratum/es2k/certs
+    cp $BASE_DIR/tls/certs/infrap4d/* /usr/share/stratum/certs
+  else
+    echo "Missing infrap4d certificates. Run \"make gen-certs\" and try again."
+    exit 1
+  fi
+}
+
+function copy_cert_to_remote() {
+  if [ -d "$BASE_DIR/tls/certs/infrap4d" ]; then
+    # copy certs to remote infrap4d dir
+    scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null $BASE_DIR/tls/certs/infrap4d/* $REMOTE_HOST:/usr/share/stratum/es2k/certs
+    check_status $? "scp infrap4d/certs/* root@$REMOTE_HOST:/usr/share/stratum/certs"
+
+    # copy certs to remote inframanager dir
+    scp -r -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null $BASE_DIR/tls/certs/inframanager/* $REMOTE_HOST:/etc/pki/inframanager/certs
+    check_status $? "scp inframanager/certs/* root@$REMOTE_HOST:/etc/pki/inframanager/certs"
+  else
+    echo "Missing infrap4d certificates. Run \"make gen-certs\" and try again."
+    exit 1
+  fi
+}
+
+function copy_script_to_remote() {
+  if [ -f "$BASE_DIR/$ARM_SCRIPT" ]; then
+    # copy acc setup script to k8s dir
+    scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null $BASE_DIR/$ARM_SCRIPT $REMOTE_HOST:/opt/p4/k8s/.
+    check_status $? "scp scripts/setup_arm_infra.sh/* root@$REMOTE_HOST:/opt/p4/k8s"
+  else
+    echo "Missing arm script"
+    exit 1
+    fi
+}
+
+function setup_run_env () {
+  $P4CP_INSTALL/sbin/copy_config_files.sh $P4CP_INSTALL $SDE_INSTALL
+  GROUP=$((${SDE_INSTALL}/bin/vfio_bind.sh 8086:1453) 2> /dev/null)
+  GROUP_ID=$(echo "$GROUP" | grep -o "Group = [0-9]*" | awk '{print $3}')
+  
+  file="/usr/share/stratum/es2k/es2k_skip_p4.conf"
+  cp $file $file.bkup
+  sed -i "s/\"iommu_grp_num\": *[0-9][0-9]*/\"iommu_grp_num\": $GROUP_ID/g" "$file"
+  sed -i "s/\(\"program-name\": \)\"[^\"]*\"/\1\"k8s_dp\"/" $file
+  sed -i "s/\(\"bfrt-config\": \)\"[^\"]*\"/\1\"\/share\/infra\/k8s_dp\/bf-rt.json\"/" $file 
+  sed -i "s/\(\"p4_pipeline_name\": \)\"[^\"]*\"/\1\"main\"/" $file
+  sed -i "s/\(\"context\": \)\"[^\"]*\"/\1\"\/share\/infra\/k8s_dp\/context.json\"/" $file
+  sed -i "s/\(\"config\": \)\"[^\"]*\"/\1\"\/share\/infra\/k8s_dp\/tofino.bin\"/" $file
+  sed -i "s/\(\"path\": \)\"[^\"]*\"/\1\"\/share\/infra\/k8s_dp\"/" $file
+}
+
+function run_infrap4d () {
+  getPid=$(pgrep -f infrap4d)  #  kill if already runnning
+  [[ $getPid ]] && kill $getPid
+  sleep 1
+  #for running infrap4d in foreground to debug
+  #gdb --args $P4CP_INSTALL/install/sbin/infrap4d -grpc_open_insecure_mode=true --nodetach
+  $P4CP_INSTALL/sbin/infrap4d &
+  check_status $? "sbin/infrap4d"
+}
+
+#############################################
+##### main ##################################
+#############################################
+
+BASE_DIR="$(dirname "$(readlink -f "$0")")"
+IF_MAX=8
+MODE="host"
+REMOTE_HOST="10.10.0.2"
```

Comment:
Same here. As the manager's IP is not static, either we should find a way to read this from somewhere or let the user know via readme.

Reply:
It's just default value. It is read from user arguments with "-r <IP ADDRESS>"

Reply:
The "-r" seems to be a mandatory argument for running in split mode. If that's true then we don't need any default, isn't it?

Reply:
Yes. This is only for guidance. I can take it out once we have detailed Readme